package.json

```diff
@@ -20,7 +20,7 @@
     "install": "NODE_ENV=production npm run production",
     "postinstall": "NODE_ENV=production npm run fetch_project",
     "specifications": "bash build.sh",
-    "build_netlify": "NODE_ENV=production npm run production && npm run specifications && npm run fetch_project",
+    "build_netlify": "NODE_ENV=production npm run production && npm run fetch_project",
```

Comment:
If we update this in conjunction with the redirect you have, then once this hits production level, this will no longer work as all of the netlify builds use this command. Maybe we should split it into `build_netlify_light` and `build_netlify`?

Reply:
Why would this no longuer work?

Reply:
I think you wrote your comment while I was still working on this.

I since changed the URL to be a relative path. The proxy works on previews and production now.

The build time went from ~5-6 minutes to 1 minute!

Reply:
Once this hits production, nothing would ever build the specs, which is what the redirects rely on

Reply:
Yeah, that is the goal! We no longer need to build the specification section for this website. It will use the content from https://github.com/jakartaee/specifications directly!

Reply:
Ahh nvm, Christopher reached out and pointed out these are different sites. D'oh!